Tests/Functional/EventListener/AutoRouteListenerTest.php

```diff
@@ -552,4 +552,40 @@ public function testConflictResolverDefaultThrowException()
         $this->getDm()->persist($blog);
         $this->getDm()->flush();
     }
+
+    public function testGenericNodeShouldBeConvertedInAnAutoRouteNode()
+    {
+        $this->assertNull($this->getDm()->find(null, '/test/auto-route/blog'));
+        $this->assertNull($this->getDm()->find(null, '/test/auto-route/blog/my-post'));
```

Comment:
We should take for granted that these do not exist. The test should be started in a consistent state.